src/components/OrgCard/index.jsx

```diff
@@ -1,11 +1,61 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import "./style.css";
+function OrgCard() {
+  const [selectedIndex, setSelectedIndex] = useState(null);
+  let bgColors = [
+    'rgba(105, 196, 105, 1)',
+    'rgba(247, 211, 68, 1)',
+    'rgba(122, 189, 244, 1)'
+  ];
 
-function index() {
-    return (
-        <div>
-            
-        </div>
-    )
+  const cardClick = index => {
+    if (selectedIndex === index) {
+      setSelectedIndex(null);
+    } else {
+      setSelectedIndex(index);
+    }
+  };
+  return (
+    <div>
+      <div className="scrollableDiv">
+        {[...Array(10)].map((_, i) => {
+          return (
+            <div
+              className={`org_card ${i === selectedIndex ? "expanded" : ""}`}
+              style={i === selectedIndex ? {backgroundColor: bgColors[Math.floor(Math.random() * 3)]}:{}}
```

Comment:
` bgColors[Math.floor(Math.random() * 3)]` have it in a different variable so the code looks cleaner

also instead of this 
```javascript
style={i === selectedIndex ? {backgroundColor: bgColors[Math.floor(Math.random() * 3)]}:{}}
```

do 

```javascript
style={{ backgroundColor: i === selectedIndex && bgColors[Math.floor(Math.random() * 3)] }}
```